addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanClassicDeviceHandler.java

```diff
@@ -118,18 +120,26 @@ public void channelUnlinked(@NonNull ChannelUID channelUID) {
     }
 
     protected boolean addListener(Channel channel) {
-        if (channel != null && channel.getChannelTypeUID().getId().startsWith("rockerswitchListener")) {
+        if (channel == null) {
+            return true;
+        }
+
+        ChannelTypeUID channelTypeUID = channel.getChannelTypeUID();
+        String id = channelTypeUID == null ? "" : channelTypeUID.getId();
+
+        if (id.startsWith("rockerswitchListener")) {
```

Comment:
Please use a constant here. There should be channel id constants already existing somewhere, just add it there so that a fellow developer immediately is less likely to break anything if he/she renames channels.

Reply:
Sure, done. Thanks!